src/containers/Card.js

```diff
@@ -74,24 +52,8 @@ const Card = props => {
               source={image}
             >
               <View style={styles.overlayContainer}>
-                {featuredTitle &&
-                  <Text
-                    style={[
-                      styles.featuredTitle,
-                      featuredTitleStyle && featuredTitleStyle,
-                    ]}
-                  >
-                    {featuredTitle}
-                  </Text>}
-                {featuredSubtitle &&
-                  <Text
-                    style={[
-                      styles.featuredSubtitle,
-                      featuredSubtitleStyle && featuredSubtitleStyle,
-                    ]}
-                  >
-                    {featuredSubtitle}
-                  </Text>}
+                {renderFeaturedTitle(featuredTitle, featuredTitleStyle, styles.featuredTitle)}
+                {renderFeaturedTitle(featuredSubtitle, featuredSubtitleStyle, styles.featuredSubtitle)}
```

Comment:
this is a bit confusing. The function is named `renderFeaturedTitle` but you are also using it to render `featuredSubtitle`.